api-inference-community/docker_images/spacy/tests/test_api.py

```diff
@@ -12,7 +12,8 @@
     #  IMPLEMENT_THIS
     # "automatic-speech-recognition": "mysample-ASR",
     # "text-generation": "mysample-gpt2",
-    "token-classification": "spacy/en_core_web_sm"
+    "token-classification": "spacy/en_core_web_sm",
```

Comment:
Just to confirm, did you run this test and did it pass?

Reply:
@osanseviero Thank you! :D I will get to these soon

Reply:
I just tested this